src/seedu/addressbook/login/Login.java

```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.login;
+
+import java.util.Map;
+import java.util.HashMap;
+
+public class Login {
+
+    private boolean loginStatus;
+
+    public static final String LOGIN_PROMPT = "Please login in order to use the address book";
+    private static final String SUCCESS = "Login Success, loading...";
+    private static final String Invalid_USERNAME_OR_PASSWORD = "Invalid username or password, please try again";
+    private static final String Invalid_FORMAT = "Please input the correct login command: login username password";
+    private Map<String, String> accounts = new HashMap<String, String>();
+
+
+    public Login(){
+        this.loginStatus = false;
+        this.accounts.put("guanlong", "12345");
+        this.accounts.put("doctorA", "doctorA");
+    }
+
+    public void setLoginStatus(boolean newStatus){
+        this.loginStatus = newStatus;
+    }
+
+    public boolean getLoginStatus(){
+        return loginStatus;
+    }
+
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.